grammar.js

```diff
@@ -694,6 +732,15 @@ module.exports = grammar(C, {
 
     // Statements
 
+    _top_level_statement: ($, original) => choice(
+      original,
+      $.co_return_statement,
+      $.co_yield_statement,
+      $.for_range_loop,
+      $.try_statement,
+      $.throw_statement,
+    ),
+
```

Comment:
None of these are top level statements in c++

Reply:
@amaanq 

... this is identical to `_non_case_statement`. I wonder, did you tweak the c grammar to add essentially the exact same thing I added this for awhile back?

What is _top_level_statement specifically describing in the c grammar?

Reply:
Hmm you're right, they are the same. I changed expression_statement for top_level_statements because binary expressions caused issues with a declaration like so:

```c
TSLanguage *(*lang_parser)(void();
```

I just assumed top_level_statement was more or less the same, seeing that there were coroutine statements in top level statements in tests

Should I remove try and throw then?